docs/reference/ingest/processors/fingerprint.asciidoc

```diff
@@ -0,0 +1,89 @@
+[role="xpack"]
+[testenv="basic"]
+[[fingerprint-processor]]
+=== Fingerprint processor
+++++
+<titleabbrev>Fingerprint</titleabbrev>
+++++
+
+Calculates a hash that can be used for content fingerprinting based on the
+content of the specified fields in a document.
+
+[[fingerprint-options]]
+.Fingerprint Options
+[options="header"]
+|======
+| Name               | Required | Default       | Description
+| `fields`           | yes      | n/a           | Array containing the names of the document
+fields whose content will be used to compute the fingerprint
```

Comment:
I'd clarify "content" here as we discussed offline. I'd also add a period for consistency.
```suggestion
| `fields`           | yes      | n/a           | Array of fields to include in
the fingerprint. For objects, the processor hashes both the field key and
value. For other fields, the processor hashes only the field value.
```